roles/kubernetes/master/templates/manifests/kube-apiserver.manifest.j2

```diff
@@ -195,6 +195,11 @@ spec:
     - mountPath: /etc/ssl/certs/ca-bundle.crt
       name: rhel-ca-bundle
       readOnly: true
+    env:
+{% for key, value in proxy_env.items() %}
```

Comment:
I guess that, if `no_proxy` key is not passed every call from apiserver will be made through the proxy and I'm not sure it's intended internally.
The `proxy_env` should be copied to another specific dict (let's say `k8s_components_proxy_env`) and updated accordingly using `kube_pods_subnet` and `kube_service_addresses`